lib/data_update_scripts/20211104200856_update_each_articles_privileged_user_point_count.rb

```diff
@@ -0,0 +1,17 @@
+module DataUpdateScripts
+  class UpdateEachArticlesPrivilegedUserPointCount
+    def run
+      # This code is "idempotent" in that it's calculating and caching a value stored elsewhere.
```

Comment:
Wondering about a guard clause to not do this if the column doesn't exist.

Reply:
you mean in case the migration had not run correctly? I have to imagine we'd have larger problems at that point but maybe you've got a corner case in mind.

Reply:
My corner case might be my lack of understanding.  For DEV.to, the column exists.  But for other sites, that column might not be there.  And if they introduce this code without that column, then things go boink.

(Then again, if they pull down the change, I believe there's some guard saying "Hey, you need to run migrations.")

Reply:
I think pending migrations are safe to assume to have run at app boot, before update scripts are enqueued by the initializer, and the migrations are run inline [by the same script](https://github.com/forem/forem/blob/main/lib/tasks/app_initializer.rake), so if they fail/raise I think you won't get that far?